### ISSUE TITLE
how to install/use without composer

### ISSUE THREAD

**Issue by kiai72** [sawarame/php-json-server#14]
Is there a way to use this on a PHP server, where i only have FTP access to ? i can't use composer.

**Comment by sawarame** [comment]
Yes, download source code below and set document root to "public" directory, You can install.
https://github.com/sawarame/php-json-server/releases/tag/0.3
Thank you.
